src/mame/mr/dribling.cpp

```diff
@@ -94,6 +98,26 @@ class dribling_state : public driver_device
 	required_region_ptr<uint8_t> m_proms;
 	required_region_ptr<uint8_t> m_gfxroms;
 
+	// sound
+	required_device<netlist_mame_logic_input_device>    m_i_pb0;
+	required_device<netlist_mame_logic_input_device>    m_i_pb1;
+	required_device<netlist_mame_logic_input_device>    m_i_pb2;
+	required_device<netlist_mame_logic_input_device>    m_i_pb3;
+	required_device<netlist_mame_logic_input_device>    m_i_pb4;
+	required_device<netlist_mame_logic_input_device>    m_i_pb5;
+	required_device<netlist_mame_logic_input_device>    m_i_pb6;
+	required_device<netlist_mame_logic_input_device>    m_i_pb7;
```

Comment:
It would be more concise if you used a `required_device_array` for these.

Reply:
Done